carina-api/src/main/java/com/qaprosoft/carina/core/foundation/api/AbstractApiMethodV2.java

```diff
@@ -38,23 +42,21 @@
     private String rsPath;
     private String actualRsBody;
 
-    public AbstractApiMethodV2(String rqPath, String rsPath, String propertiesPath) {
+    /**
+     * When this constructor is called then paths to request and expected response templates are taken from @RequestTemplatePath
+     * and @ResponseTemplatePath if present
+     */
+    public AbstractApiMethodV2() {
         super("application/json");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "application/json" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1192)